### ISSUE TITLE
Esp8266-12F can't drive OLED when using SSD1306_128X64_NONAME_SW_I2C

### ISSUE THREAD

**Issue by mogerr** [olikraus/u8g2#1428]
Can't drive SD1306 OLED when using routines likes "Hello World"

`U8X8_SSD1306_128X64_NONAME_SW_I2C u8x8(/* clock=*/ 9, /* data=*/ 10, /* reset=*/ U8X8_PIN_NONE); 
	      // Digispark ATTiny85
`

The same code can be driven when **esp8266-12E** development board with **cp2102** is used, but cannot be driven when **esp8266-12F** development board with **cp2102** or **CH340** is used. It's strange and maddening.

`scl=9, sda=10` comes as a software driver pin for OLED.

Has anyone encountered the same problem?

**Comment by olikraus** [comment]
Not sure how to continue?
Can I close this issue?

**Comment by yukiloh** [comment]
Yes, I have encountered the same problem.
When using ESP8266 and CH340, I encountered a problem of USB disconnection. After the program is uploaded, the device disconnects from the computer. And it happens periodically, constantly losing connection and reconnecting.
I'm sure this is a software issue rather than a hardware problem because I uploaded other test codes without experiencing any disconnection.
